js/src/ui/StatusIndicator/statusIndicator.js

```diff
@@ -37,6 +37,11 @@ export default class StatusIndicator extends Component {
 
   render () {
     const { id, status, title, type, tooltipPlacement } = this.props;
+    const tooltip = title.find(x => !x.isEmpty) ? (
```

Comment:
you can just supply this to `data-tip`, if it's false the tooltip won't be shown.
I tried to check if `title.length != 0` but it seems that we sometimes put empty strings there?

Also `isEmpty` is kind of fishy, I'm pretty sure it will result `undefined` for any possible builtin data type in JS.